src/Character/Controller/SmoothLocomotionController.lua

```diff
@@ -29,7 +27,7 @@ Enables the controller.
 --]]
 function SmoothLocomotionController:Enable()
     self.super:Enable()
-    VRService:SetTouchpadMode(Enum.VRTouchpad.Right,Enum.VRTouchpadMode.ABXY)
```

Comment:
Any idea if removing this has side effects for touchpad controllers like the HTC Vive? I don't want to make ABXY inaccessible for functions like jumping.

Reply:
I think I don't have much of a choice but to accept this. The primary headset this affects is the HTC Vive, and I don't have access to one anymore.